### PR TITLE
Documentation & Releases.

### PR DESCRIPTION
**Pull request by AJFire14**
- Releases Page added with release notes.
- Documentation page completed: Snippets, Commands, Observers, Engines & Permissions.
- "Switch Versions" button added to menu.
- Styleguide updated.
- Theme updated.
- URL change.
- Meta tags.
- Minor issues fixed.

- Completed requested changes.
- Sorted home page buttons.

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -15,8 +15,9 @@ <h1 class="title">
 
       <nav class="level">
           <div class="level-item has-text-centered">
-              <a class="button is-danger icon-button">Homepage</a>
-              <a class="button is-danger">Documentation</a>
+              <a class="button is-danger icon-button"> Homepage </a>
+              <a class="button is-danger icon-button"> Documentation </a>
+              <a class="button is-danger"> Releases </a>
```
Comment (TobiasFeld22) [inline 404.html]: Releases shouldn't be shown on a 404 page.

File: documentation/intro.html [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ---
 layout: largePage
 column: headers/documentation.html
-color: is-link
+color: is-dark
 title: Getting Started
-subtitle: Getting started with the Spark framework
+subtitle: Getting started with the Spark framework.
```
Comment (TobiasFeld22) [inline documentation/intro.html]: Titles & subtitles do not use dots.

File: documentation/observers.html [hunk 3]
```diff
@@ -1,11 +1,87 @@
 ---
 layout: largePage
 column: headers/documentation.html
-color: is-primary
+color: is-dark
 title: Observers
 subtitle: Observe behaviour of messages or commands.
```
Comment (TobiasFeld22) [inline documentation/observers.html]: Titles & Subtitles don't use dots

File: documentation/permissions.html [hunk 4]
```diff
@@ -1,11 +1,104 @@
 ---
 layout: largePage
 column: headers/documentation.html
-color: is-danger
+color: is-dark
 title: Permissions
 subtitle: Lock certain commands to certain people.
```
Comment (TobiasFeld22) [inline documentation/permissions.html]: Titles & Subtitles don't use dots.

File: documentation/snippets.html [hunk 5]
```diff
@@ -1,11 +1,79 @@
 ---
 layout: largePage
 column: headers/documentation.html
-color: is-warning
+color: is-dark
 title: Snippets
 subtitle: Functions you can use in the whole framework.
```
Comment (TobiasFeld22) [inline documentation/snippets.html]: Titles & Subtitles don't use dots.

File: documentation/observers.html [hunk 6]
```diff
@@ -1,11 +1,87 @@
...
+            module.exports = Observer;
+    </pre>
+        
+        
+        
+        <h1 class="title"> Methods </h1>
+        
+        <ul>
+          <li>
+              <p class="subtitle"> setType(<span class="argument"> Type </span>) </p>
```
Comment (TobiasFeld22) [inline documentation/observers.html]: To keep things uniform, i suggest keeping all subtitles to `<h4>` as the styleguide suggests.

File: _layouts/default.html [hunk 7]
```diff
@@ -57,7 +66,7 @@
     <footer class="footer">
         <div class="container">
             <div class="content has-text-centered">
-                    <a href="https://github.com/sparkbots/">Sparkbots</a> &copy; 2018 | <a href="https://github.com/TobiasFeld22/Spark">Framework</a> | <a href="https://github.com/Sparkbots/Spark-documentation/">Website</a>
+                <a href="https://github.com/sparkbots/">Sparkbots</a> &copy; 2018 | <a href="https://github.com/TobiasFeld22/Spark">Framework</a> | <a href="https://github.com/Sparkbots/Spark-documentation/">Website</a>
```
Comment (TobiasFeld22) [inline _layouts/default.html]: Update to `Sparkbots/Spark`

File: documentation/intro.html [hunk 8]
```diff
@@ -25,15 +25,15 @@ <h1 class="title">Setting up the bot</h1>
 Download a recommended code editor from below:<br><br>
```
Comment (TobiasFeld22) [inline documentation/intro.html]: Wrap this in `<p>`

File: documentation/intro.html [hunk 9]
```diff
@@ -94,15 +94,15 @@ <h1 class="title">Starting the bot</h1>
...
 When you are in the same folder as your bot file, type the command <code>npm install sparkbots</code> to install the Spark Framework. <br>
 After everything is finished, type <code>node bot.js</code> to start your bot.<br>
-If everything went correctly, you should recieve a reply on the window starting with:
+If everything went correctly, you should receive a reply on the window starting with:
 <pre>Your bot (bot-name) is now online! | Running on 1 servers
 </pre> <br>
 
 <code>bot-name</code> is the name you previously set in the application. <br> <br>
 
 You have now successfully set up your bot!<br><br>
```
Comment (TobiasFeld22) [inline documentation/intro.html]: wrap up in `<p>`

**Review by TobiasFeld22 — CHANGES_REQUESTED** [review]
I like the work that's been done so far. 
Things that i would like to bring up is that subtitles don't use dots, that the getting started page has a lot of lines without `<p>` and with a lot of `<br>` tags and that some links should be updated to reflect repository changes.

File: documentation/observers.html (hunk 6 above)
Comment (AJFire14) [reply documentation/observers.html]: Complete.

File: _layouts/default.html (hunk 7 above)
Comment (AJFire14) [reply _layouts/default.html]: Complete.

File: documentation/intro.html (hunk 8 above)
Comment (AJFire14) [reply documentation/intro.html]: Complete.

File: documentation/intro.html (hunk 9 above)
Comment (AJFire14) [reply documentation/intro.html]: Complete.

File: documentation/snippets.html (hunk 5 above)
Comment (AJFire14) [reply documentation/snippets.html]: Complete.

File: documentation/permissions.html (hunk 4 above)
Comment (AJFire14) [reply documentation/permissions.html]: Complete.

File: documentation/observers.html (hunk 3 above)
Comment (AJFire14) [reply documentation/observers.html]: Complete.

File: documentation/intro.html (hunk 2 above)
Comment (AJFire14) [reply documentation/intro.html]: Complete.

File: 404.html (hunk 1 above)
Comment (AJFire14) [reply 404.html]: Complete.

**Review by TobiasFeld22 — APPROVED** [review]

**Pull request merged**